org.eclipse.jdt.ls.core/pom.xml

```diff
@@ -36,7 +36,6 @@
 				<version>2.1.1</version>
 				<executions>
 					<execution>
-						<?m2e execute onConfiguration?>
```

Comment:
No, you want to use `<?m2e ignore?>` instead, else you'll get

> Plugin execution not covered by lifecycle configuration: org.codehaus.gmaven:groovy-maven-plugin:2.1.1:execute (execution: default, phase: generate-resources)

Reply:
addressed.